test/extensions/quic_listeners/quiche/integration/quic_http_integration_test.cc

```diff
@@ -537,5 +537,11 @@ TEST_P(QuicHttpIntegrationTest, RequestResponseWithTrailers) {
                /*response_trailers_present=*/true);
 }
 
+// Multiple 1xx before the request completes.
+TEST_P(QuicHttpIntegrationTest, EnvoyProxyingEarlyMultiple1xx) {
```

Comment:
Think it's worth a test of the reset for 101 here, or want to TODO it for later?

Reply:
It is already unit tested and will be covered by protocol integration test once HTTP3 is added into the test.

Reply:
Sorry, I misunderstood the test for non-100 case. 101 response status is indeed a HTTP3 specific case. Added an integration test here.